### ISSUE TITLE
Android TV favorites

### ISSUE THREAD

**Issue by Spp595** [realOxy/M3UAndroid#76]
Android TV has the function of adding favorite videos, movies, channels from applications.  I wish m3u had such support
![d8492c1a-8f31-11ea-b826-e2df12b27cbc](https://github.com/realOxy/M3UAndroid/assets/104263581/a198eea9-8ffe-4ac7-b686-3a8ebd5e9eaa)

The picture shows what it looks like using netflix as an example.  For example, the same function is available in vlc

**Comment by realOxy** [comment]
Thank you for your suggestion, this just solves the problem of shortcut not being generated on TV

**Comment by Spp595** [comment]
Good afternoon, issue #45 is closed, but adding favorite channels as in my example does not work, has this feature been added?

**Comment by realOxy** [comment]
Not yet, I am working on other feature requests recently. And I do not find the best practice code about adding to the tv favourite.

**Comment by Spp595** [comment]
Sadly

**Comment by oxyroid** [comment]
https://proandroiddev.com/android-tv-preview-channels-a-beginners-guide-52621502e996

**Comment by Spp595** [comment]
> [https://proandroiddev.com/android-tv-preview-channels-a-beginners-guide-52621502e996 ](https://proandroiddev.com/android-tv-preview-channels-a-beginners-guide-52621502e996)
Don't Work

**Comment by oxyroid** [comment]
No, I just leave a link to an article here so I can implement this feature later.

**Comment by sguinetti** [comment]
Hello. I find this page abour recommendations from home in Android TV: https://developer.android.com/training/tv/discovery/recommendations-channel

**Comment by sguinetti** [comment]
> Android TV has the function of adding favorite videos, movies, channels from applications. I wish m3u had such support ![d8492c1a-8f31-11ea-b826-e2df12b27cbc](https://github.com/realOxy/M3UAndroid/assets/104263581/a198eea9-8ffe-4ac7-b686-3a8ebd5e9eaa)
> 
> The picture shows what it looks like using netflix as an example. For example, the same function is available in vlc

This image coincides with the launch of the new Android TV Home in 2017: https://android-developers.googleblog.com/2017/05/welcome-to-your-new-home-on-android-tv.html
